mcs/class/System.Data/System.Data-net_4_x.csproj

```diff
@@ -49,6 +49,26 @@
   </PropertyGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <ItemGroup>
+    <Compile Include="..\..\..\external\corefx\src\System.Data.Common\src\System\Data\SQLTypes\INullable.cs" />

```

Comment:
@EgorBo ah ok, yeah that's what the bot does as well :) I usually don't run it out of convenience and let the bot handle it after the PR is merged.